### PR TITLE
BZ2094220: Add bootloader to the supported TuneD plugin list

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2094220

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/tuneD/scalability_and_performance/using-node-tuning-operator.html#supported-tuned-daemon-plug-ins_node-tuning-operator

Additional information:

QE @juzhao

### CODE REVIEW

**Review by liqcui — COMMENTED** [review]
/lgtm

File: modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc [hunk 1]
```diff
@@ -24,15 +24,20 @@ using custom profiles defined in the `profile:` section of the Tuned CR:
...
 that is not supported. The following TuneD plug-ins are currently not supported:
 
-* bootloader
 * script
 * systemd
 
 
+[WARNING]
+====
+This functionality is currently only supported on {op-system-first} 8.x worker nodes. On {op-system-base-full} 7.x worker nodes the TuneD `[bootloader]` plug-in is currently not supported.
```
Comment (darshan-nagaraj) [inline modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc:38]:
```suggestion
This functionality is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD `[bootloader]` plug-in is currently not supported.
```

Comment (darshan-nagaraj) [reply modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc:38]: "This functionality" ? Can you be more specific ?

**Review by darshan-nagaraj — COMMENTED** [review]
Small nits. Overall looks good.

File: modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc:38]: Addressed

File: modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc [hunk 2]
```diff
@@ -24,15 +24,20 @@ using custom profiles defined in the `profile:` section of the Tuned CR:
...
 that is not supported. The following TuneD plug-ins are currently not supported:
 
-* bootloader
 * script
 * systemd
 
 
+[WARNING]
+====
+The TuneD `[bootloader]` plug-in is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD `[bootloader]` plug-in is currently not supported.
```
Comment (pneedle-rh) [inline modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc:38]: Is there a specific reason to specify `[bootloader]` is square brackets in this note, but not in the bulleted list? If there is not, you could consider using a consistent format throughout.

Also, the `[bootloader]` plug-in reference in this note is in back-ticks, but the references to each of the plug-ins in the bulleted list are not in back-ticks. Are these plug-in names referencing code elements? You could update the references in the bulleted list and the node to either consistently use back-ticks or not use back-ticks.

This also applies to the note in `modules/configuring-huge-pages.adoc` that is also updated in this PR.

**Review by pneedle-rh — COMMENTED** [review]
@xenolinux I have added one minor comment to this PR. Please let me know if you have any questions. Thanks!

**Comment by pneedle-rh** [comment]
@xenolinux can you please also update the preview? The preview link is currently producing a 404. For my review, I built the preview locally. Thanks!

File: modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc]: Addressed

**Pull request merged**